app/build.gradle

```diff
@@ -95,6 +95,7 @@ android {
 
     kotlinOptions {
         jvmTarget = '1.8'
+        useIR = true
```

Comment:
I missed this change, what's the use of this?

Reply:
Here is the explanation: https://blog.jetbrains.com/kotlin/2021/02/the-jvm-backend-is-in-beta-let-s-make-it-stable-together/